runtime/compiler/ilgen/Walker.cpp

```diff
@@ -7012,6 +7186,89 @@ TR_J9ByteCodeIlGenerator::storeFlattenableInstanceWithHelper(int32_t cpIndex)
    genTreeTop(helperCallNode);
    }
 
+void
+TR_J9ByteCodeIlGenerator::storeFlattenableInstance(int32_t cpIndex)
+   {
+   /* An example on what the tree with flattened fields would look like
+    *
+    * class AssortedReftWithLongAlignment {
+    *    flattened Point2D point;
+    *    flattened Line2D line;
+    *    flattened ValueObject o;
+    *    flattened ValueLong l;
+    *    flattened ValueDouble d;
+    *    flattened ValueInt i;
+    *    flattened Triangle2D tri;
+    * }
+    *
+    * AssortedRefWithLongAlignment.setpoint(QPoint2D;)V
```

Comment:
I would suggest making the operands a little more explicit here with bytecode or some Java/pseudo-code so that it's easier to tie this to the trees that are shown in the example.